vcpkg/vcpkg.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "bsa",
+  "version-string": "2.0.3",
+  "dependencies": [
+    "lz4",
+    "zlib"

```

Comment:
```suggestion
    "lz4",
    {
      "name": "vcpkg-cmake",
      "host": true
    },
    {
      "name": "vcpkg-cmake-config",
      "host": true
    },
    "zlib"
```

Reply:
What does that do?

Reply:
This enables `vcpkg_cmake_configure`, `vcpkg_cmake_install`, and `vcpkg_cmake_config_fixup`

Reply:
I have no idea what they are, but I'll have a look, thanks